jupyterlab_gcpscheduler/src/service/gcp.ts

```diff
@@ -374,6 +446,50 @@ export class GcpService {
     }
   }
 
+  /** Polls the provided Operation at 1s intervals until it has completed. */
+  private async _pollOperation(path: string): Promise<Operation> {
```

Comment:
We use this in a couple of different places. No-op for now, but it's let add a TODO to try to factor this into a common library down the road.